Jenkinsfile

```diff
@@ -0,0 +1,124 @@
+						}
+					}
+				}
+			}
+		}
+		// This stage backs up the gk_central database before it is modified.
+		stage('Setup: Back up gk_central before modifications'){
+			steps{
+				script{
+					withCredentials([usernamePassword(credentialsId: 'mySQLCuratorUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
+						def central_before_go_update_dump = "${env.GK_CENTRAL}_${currentRelease}_before_go_update.dump"
+						sh "mysqldump -u$user -p$pass -h${env.CURATOR_SERVER} ${env.GK_CENTRAL} > $central_before_go_update_dump"
+						sh "gzip -f $central_before_go_update_dump"
+					}
+				}
+			}
+		}
+		// Download go.obo and ec2go files from GO.
+		stage('Setup: Download go.obo and ec2go files'){
+			steps{
+				script{
+					sh "wget http://current.geneontology.org/ontology/go.obo"
+					sh "wget http://current.geneontology.org/ontology/external2go/ec2go"
+					sh "mv go.obo src/main/resources/"
+					sh "mv ec2go src/main/resources/"
+				}
+			}
+		}
+		// This stage builds the jar file using maven.
+		stage('Setup: Build jar file'){
+			steps{
+				script{
+					sh "mvn clean compile assembly:single"
+				}
+			}
+		}
+		// This stage executes the GOUpdate jar file. 
+		stage('Main: GO Update'){
+			steps {
+				script{
+					withCredentials([file(credentialsId: 'Config', variable: 'ConfigFile')]){
+						sh "java -Xmx${env.JAVA_MEM_MAX}m -jar target/go-update-*-jar-with-dependencies.jar $ConfigFile"
+					}
+				}
+			}
+		}
+		// This stage backs up the gk_central and slice_current databases after they have been modified.
+		stage('Post: Backup gk_central after modifications'){
+			steps{
+				script{
+					withCredentials([usernamePassword(credentialsId: 'mySQLCuratorUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
+						def central_after_update_go_update_dump = "${env.GK_CENTRAL}_${currentRelease}_after_go_update.dump"
+						sh "mysqldump -u$user -p$pass -h${env.CURATOR_SERVER} ${env.GK_CENTRAL} > $central_after_update_go_update_dump"
+						sh "gzip -f $central_after_update_go_update_dump"
+					}
+				}
+			}
+		}
+		// This stage archives the contents of the 'reports' folder generated by GO Update and sends them in an email to the default recipients list.
+		stage('Post: Email GO Update Reports'){
+			steps{
+				script{
+					sh "tar zcf go-update-v${currentRelease}-reports.tgz reports/"
+					emailext (
+						body: "Hello,\n\nThis is an automated message from Jenkins regarding an update for v${currentRelease}. The GO Update step has completed. Please review the reports attached to this email. If they look correct, these reports need to be uploaded to the Reactome Drive at Reactome>Release>Release QA>V${currentRelease}_QA>V${currentRelease}_QA_GO_Update_Reports. If they don't look correct, please email the developer running Release. \n\nThanks!",
+						to: '$DEFAULT_RECIPIENTS',
```

Comment:
Where is `$DEFAULT_RECIPIENTS` defined?

Reply:
It is defined in the Jenkins configurations. Jenkins>Manage Jenkins>Configure System>Extended E-mail Notification>Default Recipients

Reply:
Ok. Note for future development: This step should have its own list of recipients. I would think that this GO Update step sends reports to Lisa + other curators. Other steps maybe should just send to us, so _we_ should be the `$DEFAULT_RECIPIENTS` and then this step should send to `$DEFAULT_RECIPIENTS` + Lisa + whoever else...

Reply:
Yea thats a good idea, I'll look into making separate lists that can be drawn from